src/main/kotlin/net/atos/zac/app/zaken/model/RESTZakenVerdeelGegevens.kt

```diff
@@ -13,9 +14,11 @@ import java.util.UUID
 data class RESTZakenVerdeelGegevens(
     var uuids: List<UUID>,
 
-    var groepId: String? = null,
+    var reden: String,
 
-    var behandelaarGebruikersnaam: String? = null,
+    @field:NotBlank
```

Comment:
Does this mean we no longer accept `null`? Seems this is used as a REST - will this allow us to omit these fields/keys?